docs/manage-persistent-storage.md

```diff
@@ -0,0 +1,391 @@
+- will remove unit charmed-etcd/3
+- will remove unit charmed-etcd/4
+- will remove unit charmed-etcd/5
+- will remove storage logs/6
+- will remove storage logs/7
+- will remove storage logs/8
+- will detach storage data/0
+- will detach storage data/2
+- will detach storage data/4
+
+Continue [y/N]? y
+```
+
+You can check the status after removal by running the `juju status --storage` command:
+```shell
+Model  Controller      Cloud/Region         Version  SLA          Timestamp
+etcd   dev-controller  localhost/localhost  3.6.0    unsupported  13:11:11Z
+
+Storage Unit  Storage ID  Type        Pool          Mountpoint  Size     Status    Message
+              data/0      filesystem  etcd-storage              8.0 GiB  detached  
+              data/2      filesystem  etcd-storage              8.0 GiB  detached  
+              data/4      filesystem  etcd-storage              8.0 GiB  detached  
+
+Model "admin/etcd" is empty.
+```
+
+### Deploy new cluster with existing storage
+After you removed your charmed etcd application, deploy a new one attaching the `data` volume with most recent data and 
+configure the secret URI containing the admin user's password:
+```shell
+juju deploy charmed-etcd etcd --attach-storage=data/0 --config system-users=secret:cuvh9ggv7vbc46jefvjg
+```
+
+To be able to read the password from the secret, grant secret access to the new charmed etcd application again:
+```shell
+juju grant-secret mysecret charmed-etcd
+```
+
+Watch the progress of your deployment again with `juju status --storage --watch=3s`:
+```shell
+Model  Controller      Cloud/Region         Version  SLA          Timestamp
+etcd   dev-controller  localhost/localhost  3.6.0    unsupported  13:25:01Z
+
+App           Version  Status  Scale  Charm         Channel  Rev  Exposed  Message
+charmed-etcd           active      1  charmed-etcd             3  no       
+
+Unit             Workload  Agent  Machine  Public address  Ports  Message
+charmed-etcd/8*  active    idle   8        10.105.253.225         
+
+Machine  State    Address         Inst id        Base          AZ  Message
+8        started  10.105.253.225  juju-84b57b-8  ubuntu@24.04      Running
+
+Storage Unit    Storage ID  Type        Pool          Mountpoint                                  Size     Status    Message
+                data/2      filesystem  etcd-storage                                              8.0 GiB  detached  
+                data/4      filesystem  etcd-storage                                              8.0 GiB  detached  
+charmed-etcd/8  data/0      filesystem  etcd-storage  /var/snap/charmed-etcd/common/var/lib/etcd  8.0 GiB  attached  
+charmed-etcd/8  logs/12     filesystem  rootfs        /var/snap/charmed-etcd/common/var/log/etcd  76 GiB   attached  
+```
+
+Once the application is in active status, scale up attaching the other volumes:
+```shell
+juju add-unit charmed-etcd --attach-storage=data/2
+juju add-unit charmed-etcd --attach-storage=data/4
+```
```

Comment:
Similar note on reminding the user that the data in these volumes will be overwritten